nucypher-core-wasm-bundler/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nucypher/nucypher-core",
-  "version": "0.14.1",
+  "version": "0.14.5",
```

Comment:
As an aside: @KPrasch , `nucypher` is still using `nucypher-core` `0.13.0` - https://github.com/nucypher/nucypher/blob/v7.4.x/pyproject.toml#L9. Is it time to update that? From the `nucypher-core` changelog, there doesn't seem to be a compatibility problem for nodes.

Reply:
I don't think there are any breaking of functional issues. IIRC, the difference between `0.13.x` and `0.14.x` is only related to WASM bindings.

Reply:
Speaking of the changelog - it could be good to update it for versions 0.14.1 -> 0.14.5 even if it is to say "Fixed packaging issue".